components/mpas-ocean/src/analysis_members/mpas_ocn_moc_streamfunction.F

```diff
@@ -507,10 +507,12 @@ subroutine ocn_compute_moc_streamfunction(domain, timeLevel, err)!{{{
                      * 0.5_RKIND*(layerThickness(k,c1) + layerThickness(k,c2))
                end do
             end do
-            do k = 2, nVertLevels
+            mocStreamValLatAndDepthRegionLocal(1, nVertLevels, iTransect) = &
+                  - sumTransport(nVertLevels, iTransect)
+            do k = nVertLevels-1,1,-1
                mocStreamValLatAndDepthRegionLocal(1, k, iTransect) = &
-                  mocStreamValLatAndDepthRegionLocal(1, k - 1, iTransect) &
-                  + sumTransport(k - 1, iTransect)
+                  mocStreamValLatAndDepthRegionLocal(1, k + 1, iTransect) &
+                  - sumTransport(k + 1, iTransect)
```

Comment:
Note that this change came from a discussion with Gokhan about how to calculate AMOC.  If preferred I can make this one change its own PR.

Reply:
Sorry for being nosy, but how do the old and new way of computing AMOC affect the AMOC curves you show above?

Reply:
No problem at all.  Thanks for looking.  The change ends up being about 0.5-1 Sv on average increase, but not every month, some months go up some down.

Reply:
So summing from the bottom up?  Could you explain a bit more about the k index change?  I don't fully follow how the sign change and the index change relate to each other.  Does the sign of the resulting MOC somehow remain the same?

Reply:
I think it would be better to make this its own PR and explain the change further.  We would want to make the corresponding change in MPAS-Analysis and do some tests to make sure the two are consistent.